recipe/meta.yaml

```diff
@@ -152,15 +152,19 @@ outputs:
         - ray.autoscaler
 
   - name: ray-dashboard
+    script: cd python/ray/new_dashboard/client && npm install && npm ci && npm run build
     requirements:
       host:
+        - nodejs
         - python
       run:
         - python
         - {{ pin_subpackage('ray-core', exact=True) }}
     test:
       imports:
         - ray.new_dashboard
+      commands:
+        - python -c "import ray; ray.init()"
```

Comment:
Testing like this will simply print a warning and carry on. The test will always pass. To get the test to fail if dashboard is missing this needs to change to:
```
python -c "import ray; ray.init(include_dashboard=True)"
```

Reply:
Thanks for the tip/help!